x-pack/test/api_integration/config.js

```diff
@@ -7,7 +7,9 @@
 import { services } from './services';
 
 export async function getApiIntegrationConfig({ readConfigFile }) {
-  const xPackFunctionalTestsConfig = await readConfigFile(require.resolve('../functional/config.js'));
+  const xPackFunctionalTestsConfig = await readConfigFile(
+    require.resolve('../functional/config.js')
```

Comment:
Looks like you might be using the prettier plugin in your editor, which is going to cause random edits like this because we don't configure prettier to know which files should/shouldn't be prettierified, that's done with eslint. If that's the case please disable the prettier plugin and rely just on eslint instead, it applies prettier to the correct files and can be configured to apply to more files here: https://github.com/elastic/kibana/blob/master/.eslintrc.js#L220-L258

Reply:
@spalger thanks for the tip

Reply:
I disabled prettier and restored the rando changes I introduced.